.travis.yml

```diff
@@ -27,6 +27,7 @@ cache:
   - $HOME/.pip-cache
 
 install:
+  - pip install .
```

Comment:
that is for installing the console scripts in the environment, so the tests can execute `luigid` (instead of `./bin/luigid`)